website/docs/r/extended_location_custom_location.html.markdown

```diff
@@ -0,0 +1,138 @@
+    ENV = "Test"
+  }
+}
+
+resource "local_file" "example" {
+  depends_on = [
+    azurerm_kubernetes_cluster.example,
+  ]
+  content  = azurerm_kubernetes_cluster.example.kube_config_raw
+  filename = "${path.module}/kubeconfig-example"
+}
+
+resource "null_resource" "example1" {
+  depends_on = [
+    azurerm_kubernetes_cluster.example,
+  ]
+  provisioner "local-exec" {
+    command = "set KUBECONFIG=${path.module}/kubeconfig-example"
+  }
+}
+
+resource "null_resource" "example2" {
+  depends_on = [
+    null_resource.example1,
+  ]
+  provisioner "local-exec" {
+    command = "az connectedk8s connect --name example-akcc --resource-group ${azurerm_resource_group.example.name} --subscription ${data.azurerm_client_config.current.subscription_id} --location ${azurerm_resource_group.example.location} --kube-config=${path.module}/kubeconfig-example --kube-context example"
+  }
+}
+
+resource "azurerm_arc_kubernetes_cluster_extension" "example" {
+  name           = "example-ext"
+  cluster_id     = azurerm_arc_kubernetes_cluster.example.id
+  extension_type = "microsoft.flux"
+}
+
+resource "azurerm_extended_location_custom_location" "example" {
+  name                = "example-custom-location"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = "West Europe"
+  cluster_extension_ids = [
+    azurerm_arc_kubernetes_cluster_extension.example.id
+  ]
+  display_name     = "example-custom-location"
+  namespace        = "example-namespace"
+  host_resource_id = azurerm_arc_kubernetes_cluster.example.id
+  authentication {
+    value = base64encode(azurerm_kubernetes_cluster.example.kube_config_raw)
+  }
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Specifies the name which should be used for this Custom Location. Changing this forces a new Custom Location to be created.
+
+* `resource_group_name` - (Required) Specifies the name of the Resource Group where the Custom Location should exist. Changing this forces a new Custom Location to be created.
+
+* `location` - (Required) Specifies the Azure location where the Custom Location should exist. Changing this forces a new Custom Location to be created.
+
+* `namespace` - (Required) Specifies the namespace of the Custom Location. Changing this forces a new Custom Location to be created.
+
+* `cluster_extension_ids` - (Required) Specifies the list of Cluster Extension IDs.
+
+* `host_resource_id` - (Required) Specifies the host resource ID.
+
+* `authentication` - (Optional) An `authentication` block as defined below.
+
+* `display_name` - (Optional) Specifies the display name of the Custom Location.
+
+* `host_type` - (Optional) Specifies the host type of the Custom Location. The only possible values is `KubernetesCluster`.
+
+---
+
+An `authentication` block supports the following:
+
+* `type` - (Optional) Specifies the type of authentication.
+
+* `value` - (Required) Specifies the value of authentication.
```

Comment:
We need some possible values here to help users complete their configuration.

Reply:
The example code above is updated with this info.